assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -303,21 +309,78 @@ const useQuickActions = () => {
     ]
   );
 
+  const backgroundElementMediaActions = useMemo(() => {
+    const resetProperties = getResetProperties(
+      selectedElement,
+      selectedElementAnimations
+    );
+
+    return [
+      {
+        Icon: PictureSwap,
+        label: ACTION_TEXT.REPLACE_BACKGROUND_MEDIA,
+        onClick: handleFocusMediaPanel(selectedElement?.id),
+        ...actionMenuProps,
+      },
+      {
+        Icon: CircleSpeed,
+        label: ACTION_TEXT.ADD_ANIMATION,
+        onClick: handleFocusAnimationPanel(selectedElement?.id),
+        ...actionMenuProps,
+      },
+      {
+        Icon: Eraser,
+        label: ACTION_TEXT.CLEAR_ANIMATION_AND_FILTERS,
+        onClick: () =>
+          handleClearAnimationsAndFilters({
+            elementId: selectedElement?.id,
+            resetProperties,
+            elementType: ELEMENT_TYPE.BACKGROUND,
+          }),
+        separator: 'top',
+        disabled: resetProperties.length === 0,
+        ...actionMenuProps,
+      },
+    ];
+  }, [
+    selectedElement,
+    selectedElementAnimations,
+    handleFocusMediaPanel,
+    actionMenuProps,
+    handleFocusAnimationPanel,
+    handleClearAnimationsAndFilters,
+  ]);
+
   // Hide menu if there are multiple elements selected
   if (selectedElements.length > 1) {
     return [];
   }
 
+  const isBackgroundElementMedia = Boolean(
+    backgroundElement && backgroundElement?.resource
+  );
+
   // Return the base state if:
   //  1. no element is selected
-  //  2. the selected element is the background element
+  //  2. the selected element is the background element and it's not media
   if (
-    (selectedElements.length === 0 && backgroundElement) ||
-    selectedElements[0]?.isBackground
+    (selectedElements.length === 0 &&
+      backgroundElement &&
+      !isBackgroundElementMedia) ||
+    (selectedElements[0]?.isBackground && !isBackgroundElementMedia)
```

Comment:
💭 this could be simplified a bit but it's not necessarily more readable
```suggestion
      !isBackgroundElementMedia &&
      ((selectedElements.length === 0 &&
       backgroundElement) ||
       selectedElements[0]?.isBackground)
```

Reply:
oh i like it!